+++
title = "LncATLAS database for subcellular localisation of long noncoding RNAs"
date = 2017-09-01T00:00:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Mas-Ponte, David" , "Carlevaro-Fita, Joana" , "Palumbo, Emilio" , "Hermoso Pulido, Toni" , "Guigo, Roderic" , "Johnson, Rory"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "LncATLAS database for subcellular localisation of long noncoding RNAs"
publication_short = "LncATLAS"

# Abstract and optional shortened version.
abstract = "The subcellular localisation of long noncoding RNAs (lncRNAs) holds valuable clues to their molecular function. However, measuring localisation of newly-discovered lncRNAs involves time-consuming and costly experimental methods. We have created LncATLAS, a comprehensive resource of lncRNA localisation in human cells based on RNA-sequencing datasets. Altogether, 6,768 GENCODE-annotated lncRNAs are represented across various compartments of 15 cell lines. We introduce Relative concentration index (RCI) as a useful measure of localisation derived from ensemble RNAseq measurements. LncATLAS is accessible through an intuitive and informative webserver, from which lncRNAs of interest are accessed using identifiers or names. Localisation is presented across cell types and organelles, and may be compared to the distribution of all other genes. Publication-quality figures and raw data tables are automatically generated with each query, and the entire dataset is also available to download. LncATLAS makes lncRNA subcellular localisation data available to the widest possible number of researchers. It is available at lncatlas.crg.eu."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename of your project in `content/project/`.
#   Otherwise, set `projects = []`.
projects = ["example-external-project.md"]

# Links (optional).
url_pdf = "http://rnajournal.cshlp.org/content/23/7/1080.full.pdf+html"
url_preprint = "https://www.biorxiv.org/content/early/2017/03/22/116335.full.pdf"
#url_code = "#"
#url_dataset = "#"
#url_project = "#"
#url_slides = "#"
#url_video = "#"
#url_poster = "#"
#url_source = "#"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
url_custom = [{name = "lncATLAS Link", url = "http://lncatlas.crg.eu"}]

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = "headers/bubbles-wide.jpg"
caption = "My caption :smile:"

+++

More detail can easily be written here using *Markdown* and $\rm \LaTeX$ math code.
