---
title: "Benchmarking mutational context functions"
output: html_notebook
---


```{r}
library(SomaticCancerAlterations)
library(BSgenome.Hsapiens.1000genomes.hs37d5)
library(VariantAnnotation)

genome = BSgenome.Hsapiens.1000genomes.hs37d5
```



```{r}
sca_data = unlist(scaLoadDatasets())

sca_data$study = factor(gsub("(.*)_(.*)", "\\1", toupper(names(sca_data))))
sca_data = unname(subset(sca_data, Variant_Type %in% "SNP"))
sca_data = keepSeqlevels(sca_data,
                         SomaticSignatures::hsAutosomes(),
                         pruning.mode = "coarse")

sca_vr = VRanges(
  seqnames = seqnames(sca_data),
  ranges = ranges(sca_data),
  ref = sca_data$Reference_Allele,
  alt = sca_data$Tumor_Seq_Allele2,
  sampleNames = sca_data$Patient_ID,
  seqinfo = seqinfo(sca_data),
  study = sca_data$study)

```

```{r}
library(SomaticSignatures)
sca_motifs = SomaticSignatures::mutationContext(sca_vr,ref = genome )
res.SS = SomaticSignatures::motifMatrix(sca_motifs,normalize = FALSE)
```

```{r}
library(MutationalPatterns)
ref_genome <- "BSgenome.Hsapiens.1000genomes.hs37d5"
vcf =GRanges(sca_vr)
vcf$REF = ref(sca_vr)
vcf$ALT = alt(sca_vr)
vcf$sampleNames = sampleNames(sca_vr)
vcfs = split(vcf,vcf$sampleNames)
res.MP <- t(mut_matrix(vcf_list = vcfs[1:2],
                        ref_genome = ref_genome))
```


```{r}

library(genomicHelpersDMP)
res.DMP = compute_MSM_fast(vr = sca_vr,
                 genome = genome)
```



```{r}

bm = microbenchmark::microbenchmark(
  SomaticSignatures = {
    sca_motifs = SomaticSignatures::mutationContext(sca_vr,ref = genome )
    res.SS = SomaticSignatures::motifMatrix(sca_motifs,normalize = FALSE)
  },
  MutationalPatterns = {
    ref_genome <- "BSgenome.Hsapiens.1000genomes.hs37d5"
    vcf =GRanges(sca_vr)
    vcf$REF = ref(sca_vr)
    vcf$ALT = alt(sca_vr)
    vcf$sampleNames = sampleNames(sca_vr)
    vcfs = split(vcf,vcf$sampleNames)
    res.MP <- t(mut_matrix(vcf_list = vcfs,
                           ref_genome = ref_genome))
  },
  genomicHelpersDMP = {
    res.DMP = compute_MSM_fast(vr = sca_vr,
                               genome = genome)
  },times = 10
)

plot(bm)
```

